spec/grape/api_spec.rb

```diff
@@ -1456,6 +1456,34 @@ def custom_error!(name)
       expect(last_response.body).to eq 'hello bob'
     end
 
+    it 'avoids polluting global namespace' do
+      class A < Grape::API
```

Comment:
These should be `let(:A) { Class.new(...) }` to avoid polluting (see what I did there :) the global namespace.